### PR TITLE
docs: add code jam to meetings

### PR DESCRIPTION
**Pull request by shaneutt**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

The code jam meeting has been regularly attended by several people
since it's inception. Given that success this patch makes it more
official by advertising it on our meetings page.

### CODE REVIEW

File: site-src/contributing/index.md [hunk 1]
```diff
@@ -35,29 +35,64 @@ questions, discussions.
...
+
+The Gateway API "Code Jam" is less of a meeting and more of a hangout to discuss
+and pair on Gateway API and technologies relevant to the project. This is an
+open agenda meeting (feel free to bring your topics) where the following kinds
+of activities (focusing on Gateway API related things) are encouraged:
+
+- brainstorming
+- code pairing
+- demos
+- getting help
```
Comment (robscott) [inline site-src/contributing/index.md:82]: Trivial nit: Other lists all start with capitalized letters

**Review by robscott — COMMENTED** [review]
Thanks @shaneutt! Adding optional hold if you want to address my tiny nit, but also fine to ignore and remove hold.

/lgtm
/hold

File: site-src/contributing/index.md (hunk 1 above)
Comment (shaneutt) [reply site-src/contributing/index.md]: f763512c8d4fbc4ff427719e514b5b442f3cd5d5 fixes :+1:

**Comment by robscott** [comment]
Thanks!

/lgtm

**Pull request merged**